src/mongo_logger.ts

```diff
@@ -292,22 +306,23 @@ function attachConnectionFields(
 }
 
 function defaultLogTransform(
-  logObject: LoggableEvent | Record<string, any>
+  logObject: LoggableEvent | Record<string, any>,
+  maxDocumentLength: number = DEFAULT_MAX_DOCUMENT_LENGTH
```

Comment:
We always will default this in the options do we need to redo that here and in truncation?

Reply:
Do we have a preference for where we default it? I would skew to doing it in the log transform/logger itself

Reply:
The logger class seems good